cli/cdsctl/template_bulk.go

```diff
@@ -53,6 +53,11 @@ var templateBulkCmd = cli.Command{
 			Name:  "track",
 			Usage: "Wait the bulk to be over",
 		},
+		{
+			Type:  cli.FlagBool,
+			Name:  "parallel",
+			Usage: "Apply template on workflow in parallel",
+		},
 	},
 }
 
```

Comment:
is there now a "non interactive" mode for this command ?

Reply:
yes and output was fixed in a previous PR